src/lib/list/list.spec.ts

```diff
@@ -144,6 +145,14 @@ describe('MatList', () => {
     items.forEach(item => expect(item._isRippleDisabled()).toBe(true));
   });
 
+  it('action list should be created successfully', () => {
```

Comment:
Looks like the grammar wouldn't make sense here when viewing the test on Karma.

This would make more sense:

```typescript
it('should create an action list', () => {})
```